doc/engineering-reference/src/simulation-models-encyclopedic-reference-004/setpoint-managers.tex

```diff
@@ -70,25 +70,43 @@ \subsection{Outdoor Air Reset}\label{outdoor-air-reset}
 {T_{set,atOAlow}} = {T_{1,set,atOAlow}}
 \end{equation}
 
-\({T_{set,atOAhigh}} = {T_{1,set,atOAhigh}}\) .
+\begin{equation}
```

Comment:
The changes starting here that show different equations based on control variable type are not needed.  This section should have no changes.